javascript.html.markdown

```diff
@@ -323,6 +384,19 @@ setTimeout(function(){
     // this code will be called in 5 seconds' time
 }, 5000);
 
+// You can also write anonymous functions using arrow syntax in ES6
+// environments. The following is the same as the above.
```

Comment:
Well, no. They're not the same. Arrow functions do not introduce a new context, while `function`s do.